server/restapi/configure_down_to_meet.go

```diff
@@ -111,9 +118,3 @@ func setupMiddlewares(handler http.Handler) http.Handler {
 	c.Log = log.WithField("source", "cors")
 	return impl.RequestMiddleware(Impl.SessionMiddleware(c.Handler(handler)))
```

Comment:
Can you just move the CORS handler and impl.RequestMiddleware here, but leave Impl.SessionMiddleware in setupMiddlewares?

The CORS handler has to be in setupGlobalMiddleware, that's fine. impl.RequestMiddleware is for logging, so it's fine too. SessionMiddleware is only used for the routes, so should stay in the original function.